soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4418,6 +4419,15 @@ void CreateScrubMessages() {
     }
 }
 
+void CreateNaviRandoMessages() {
+    CustomMessageManager* customMessageManager = CustomMessageManager::Instance;
+    customMessageManager->AddCustomMessageTable(Randomizer::NaviRandoMessageTableID);
+    customMessageManager->CreateMessage(Randomizer::NaviRandoMessageTableID, 0x00,
+                                        { TEXTBOX_TYPE_BLACK, TEXTBOX_POS_BOTTOM, "Test Eng0", "Test Ger", "Test Fre" });
+    customMessageManager->CreateMessage(Randomizer::NaviRandoMessageTableID, 0x01,
+                                        { TEXTBOX_TYPE_BLACK, TEXTBOX_POS_BOTTOM, "Test Eng1", "Test Ger", "Test Fre" });
+}
+
```

Comment:
Yeah this is on the right track. One thing you can do is create an array of CustomMessageMinimal structs (see `CustomMessageManager.h` for that struct) and iterate over it. That way you at least don't have to write out `TEXTBOX_TYPE_BLACK` and `TEXTBOX_POS_BOTTOM` every time. The function that registers GetItem functions does this, if you want to check it out. Someday we'll have a way to import messages from a JSON file, but that's gonna take a while to build out.

Reply:
Got the array set up now and properly iterating through the 15 sample messages 👍